package.json

```diff
@@ -36,6 +38,7 @@
         "@vitejs/plugin-react-swc": "^3.5.0",
         "actions": "^1.3.0",
         "ci": "^2.3.0",
+        "cypress": "^13.7.3",
```

Comment:
can you integrate husky please as follows..

https://www.dhiwise.com/post/husky-react-apps-supercharge-your-react-app-code-quality
https://levelup.gitconnected.com/full-guide-add-husky-to-your-react-project-e049935f20d5

Reply:
some of the command such as 
`npx husky install ` and `npx husky add` were deprecated and didn't work.

I found this article instead

https://medium.com/@abpeter14/how-to-install-commitlint-husky-2024-f1157f14006f